scripts/bb-test-prepare.sh

```diff
@@ -22,6 +22,13 @@ SUDO="sudo -E"
 
 set -x
 
+# Attempt to blacklist the buildslave to prevent it
+# from being targeted by the oom-killer
```

Comment:
How about something more along the lines of, "Attempting to set oom_score_adj for buildslave to prevent...".